frontend-react/src/hooks/filters/UseFilterManager.ts

```diff
@@ -1,33 +1,49 @@
-import useDateRange, { DateRange } from "./UseDateRange";
-import useSortOrder, { Sort } from "./UseSortOrder";
-import usePageSize, { PageSize } from "./UsePageSize";
+import { Dispatch, useCallback } from "react";
 
-export interface FilterState {
-    range: DateRange;
-    sort: Sort;
-    pageSize: PageSize;
-}
+import useDateRange, {
+    RangeSettings,
+    RangeSettingsActionType,
+} from "./UseDateRange";
+import useSortOrder, {
+    SortSettingsAction,
+    SortSettingsActionType,
+    SortSettings,
+} from "./UseSortOrder";
+import usePages, {
+    PageSettings,
+    PageSettingsAction,
+    PageSettingsActionType,
+} from "./UsePages";
```

Comment:
Does it feel weird to you to have to _mix_ how these are imported? 

`import useDateRange, { RangeSettings, RangeSettingsActionType} from`
vs
`import { useDateRange, RangeSettings, RangeSettingsActionType} from`

Reply:
I'm impartial honestly; I kinda like the idea of the core functionality being exported as a default and any supporting types and enums following it (i.e. option 1) but option 2 also looks clean, though I'm not sure if our imports get alphepbetically sorted, but that might bury the main export in the list of supporting exports. Up for either, but I've been trying this pattern on my side project and it snuck its way into this :P

Reply:
Yeah, it's just a style thing. I wonder which approach will win out over time. 
<resolved>